oemof_industry/mimo_converter.py

```diff
@@ -150,6 +150,12 @@ def __init__(
         # Add emissions to outputs (as they are excluded from output groups before)
         outputs = reduce(operator.ior, self.output_groups.values(), {}) | emissions
 
+        # If primary is not set, define it as first output, otherwise search for primary string in inputs and outputs
+        buses_holding_investment = [bus for bus, flow in (inputs | outputs).items() if flow.investment is not None]
+        if len(buses_holding_investment) > 1:
+            raise ValueError("Only one investment allowed.")
+        self.primary_bus = buses_holding_investment[0] if len(buses_holding_investment) == 1 else None
```

Comment:
In the above code block, do you want to use `primary` parameter? I think the comment does not match what's being done in the code block.